src/main/dynamo/Table.ts

```diff
@@ -12,14 +12,18 @@ export class Table {
   constructor(config: {
     name: string
     partitionKeyName: string
-    sortKeyName: string
+    sortKeyName: string,
+    encryptionBlacklist?: string[]
   }) {
     this.tableName = config.name
     this.partitionKeyName = config.partitionKeyName
     this.sortKeyName = config.sortKeyName
 
     this.addToEncryptionBlacklist(this.partitionKeyName)
     this.addToEncryptionBlacklist(this.sortKeyName)
+
+    const encryptionBlacklist = config.encryptionBlacklist || []
+    encryptionBlacklist.forEach(this.addToEncryptionBlacklist)
```

Comment:
We can just move the initial assignment of `encryptionBlacklist` to here, eg: 

```TypeScript
  private encryptionBlacklist = new Set<string>(["model", "__jayz__metadata"])
```

Becomes 
```TypeScript
  private encryptionBlacklist: Set<string>
constructor (...) {
   ... 
  encryptionBlacklist = new Set<string>(["model", "__jayz__metadata", ...encryptionBlacklist])
}
```

Reply:
Gotcha, I like this